modules/ROOT/pages/getting-started/toolbox.adoc

```diff
@@ -12,58 +12,68 @@ With it, you can:
 . Build the Neo4j GraphQL schema.
 . Experiment, query, and play.
 
+
 == Connect to a Neo4j database
 
 Before you start using the Toolbox, make sure you have followed all xref:index.adoc#_requirements[requirements] to run a Neo4j database.
 You can use https://neo4j.com/docs/desktop-manual/current/[Neo4j Desktop] or https://neo4j.com/docs/aura/auradb/[Neo4j AuraDB] for that.
 
+
 == Set the type definitions
 
-. Set your type definitions directly in the Toolbox editor or introspect the Neo4j database you connected to.
-+
-If you followed the xref:getting-started/index.adoc[Getting started tutorial], you should see these type definitions in the GraphQL Toolbox:
-+
+Set your type definitions directly in the Toolbox editor or introspect the Neo4j database you connected to.
+
+If you followed the xref:getting-started/graphql-self-hosted.adoc[self-hosted GraphQL tutorial], use **Introspect** to see these type definitions in the GraphQL Toolbox:
+
 [source, graphql, indent=0]
 ----
-type Actor @node {
-  actedInMovies: [Movie!]! @relationship(type: "ACTED_IN", direction: OUT)
-  name: String!
+type Product @node {
+    productName: String
+    category: [Category!]! @relationship(type: "PART_OF", direction: OUT)
 }
 
-type Movie @node {
-  actorsActedIn: [Actor!]! @relationship(type: "ACTED_IN", direction: IN)
-  title: String!
+type Category @node {
+    categoryName: String
+    products: [Product!]! @relationship(type: "PART_OF", direction: IN)
 }
 ----
-+
+
 image::toolbox-schema-view.png[GraphQL Toolbox schema view]
 
-. Click the button "Build schema" and then go to the Query editor tab. 
+[NOTE]
+====
+If you modified your database after following the xref:getting-started/graphql-self-hosted.adoc[self-hosted GraphQL tutorial], you may see different introspection results.
+====
 
-. Query the Neo4j database using the autogenerated GraphQL queries and mutations from the `@neo4j/graphql` library:
+
+== Build the schema and query the database
+
+**Build schema** which takes you to **Query editor** tab. 
+
+Query the Neo4j database. Paste the following or interact with the autogenerated GraphQL queries and mutations from the `@neo4j/graphql` library from the **Explorer**:
 +
```

Comment:
this `+` is actually rended
<img width="902" height="297" alt="image" src="https://github.com/user-attachments/assets/a635ab3f-c0b7-4fb8-ba31-0fffe1ce1467" />